drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/ChromeDriverBinaryHandler.java

```diff
@@ -74,16 +74,16 @@ protected String getExpectedKeyRegex(String requiredVersion, String directory) {
         public String getFileNameRegexToDownload(String version) {
             StringBuilder fileName = new StringBuilder("chromedriver_");
             if (PlatformUtils.isMac()) {
-                fileName.append("mac64");
+                fileName.append("mac");
             } else if (PlatformUtils.isWindows()) {
-                fileName.append("win32");
+                fileName.append("win");
             } else if (PlatformUtils.isUnix()) {
                 fileName.append("linux");
-                if (PlatformUtils.is32()) {
-                    fileName.append("32");
-                } else {
-                    fileName.append("64");
-                }
+            }
+            if (PlatformUtils.is32()) {
+                fileName.append("32");
+            } else {
+                fileName.append("64");
             }
             return fileName.append(".zip").toString();
         }
```

Comment:
There is a similar one in `EdgeDriverBinary` but with a small difference in the order of the code and changing "chromedriver" with "edgedriver" could it be a joint method with the driver as another parameter in `XmlStorageSource`?

Reply:
I'm not sure, I think this should be kept per driver binary basis. In my eyes it's going a bit too far with "reuse", but I will think it over. Thanks for spotting that!

Reply:
Okay, or maybe if it is not reused they could have the same code, the condition are in a different order (althought the result is the same, but to have the same approach on both)

Reply:
Difference here is however that in case of Edge Driver there is no such thing as `mac32` thus for Mac it's always rolling back to `mac64.zip` suffix. This part (apart from filename prefix) makes those implementations different. I think we should just leave it as is.